resources/assets/components/InboxItem/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { map } from 'lodash';
+
+class InboxItem extends React.Component {
+  displayImage(photo_url) {
+    if (photo_url == "default") {
```

Comment:
it is an actual string in the database! that is how we decided to deal with broken images so we can deal with displaying them however we want